website/blog/2022-04-08-adding-fonts.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Adding fonts
+description: An intro to adding fonts to your project.
+slug: adding-fonts
+authors: iobrado
+date: 2022-04-12
+tags: [eightshift, boilerplate, fonts]
+hide_table_of_contents: false
+---
+
+Each project is different in its way. Logo, various fonts, etc. are what define the visual identity of your website. In this post, we'll cover adding fonts to a project.
+<!--truncate-->
+
+### Importing fonts into your project
+[Our documentation](/docs/basics/fonts) covers the required steps to add a font to your project, but here we'll cover the process in a bit more detail. To even start, we need a font (or two). For this example, I'll use **_SourceSansPro_** and **_NotoSerif_** downloaded from [Google Fonts](https://fonts.google.com/). These are in `.ttf` format, so convert them to `.woff` and `.woff2` however you can. We recommend the following tools:
+- https://convertio.co/ttf-woff/
+- https://cloudconvert.com/ttf-to-woff
+- https://everythingfonts.com/ttf-to-woff
+
+Fonts should go inside the **_/assets/fonts_** folder inside your theme. Paste the fonts you want to use there. You will also notice that this folder contains an **_index.js_** file. This file is used to import fonts into your project. Here's an example of how I imported my fonts:
+```js
+// SourceSansPro WOFF
+import './SourceSansPro-Bold.woff';
```

Comment:
Maybe we should not mention WOFF, because it's only there to fallback for IE11, all the other browser support WOFF2 😄

Reply:
Maybe just add a line explaining that if you don't need to support IE11, don't include `.woff` files to save bandwidth.

Reply:
@gabriel-glo I guess the latest Boilerplate and this whole article wouldn't work with IE11 anyways (because CSS variables), but I guess just noting that `.woff` can be left out is fine 😄